### ISSUE TITLE
Command blocks and /tpj Command don't work

### ISSUE THREAD

**Issue by sjaustad** [maruohon/justenoughdimensions#43]
Minecraft Version: **1.12.2**
Just Enough Dimensions Mod Version: **1.5.5**

The command block as the following command set:
/tpj @p 12

Expected Behavior: Teleports executing player to the default spawn area of dimension 12.

Actual Behavior: Nothing. No errors and the server console doesn't even say the command was run at all.

This command works exactly as copied from the chat but does not work with a button on a command block. Command blocks are enabled on the server and I can use other commands such as /tp and /say.

I have also tried
/tpj @s 12
/tpj @a 12
/tpj 12
/tpj @p 0 0 0 0
/tpj 0 @p 0 0 0 (just in case, but this syntax isn't correct)

Any help would be greatly appreciated.

**Comment by maruohon** [comment]
Hmm, well that is really strange then, because it's certainly supposed to work, and when I tried it just now, it does work in my test world. So `tpj @p 1` in a command block teleports me to the End, and `tpj @p 0` in a command block in the End teleports me back to the overworld. What does it say in the command block's Previous Output field when you try to run the tpj command?

Also in general I'd highly recommend using the latest 1.6.0 version instead of the old 1.5.5, but that's besides this issue... I tested it in 1.5.5 just now.

**Comment by sjaustad** [comment]
@maruohon 
I used the `/jed list-registered-dimensions` and it came back with `0, -1, 1, 2, 12` and obviously I can teleport to it just fine. I actually already updated to 1.6.0 to see if that would fix it but it didn't.

I'm running a [CatServer](https://github.com/CatServer/CatServer) so I can use both Forge and Bukkit mods. Is there a possibility of conflicts with any of these:

Forge Mods:
-Applied Energistics 2
-Buildcraft
-ComputerCraft
-EnderCore
-EnderIO
-FloodLights
-Inventory Tweaks
-Just enough items
-Just enough dimensions
-Magical Crops
-Not enough Items
-Project red (all of them)
-Railcraft
-SteveCarts
-WorldEdit

Plugins
-ChestShop
-iConomy
-LuckPerms
-Towny (and chat, NameUpdater)
-Vault

**Comment by maruohon** [comment]
I don't see how it would be a conflict with another Forge mod, they don't usually do stuff that would break that kind of things. However a bukkit/spigot/etc. server especially with some protection plugins might add/change the command permission such that you would need to add a permission node somewhere to run that command, maybe? That's my best and only guess as to why it wouldn't work. Does it say anything in the command block's Previous Output field after trying to execute the tpj command?

**Comment by sjaustad** [comment]
No, there is nothing in the previous outfield box. I'll look into the permissions though.

**Comment by sjaustad** [comment]
I tried removing all of the permission plugins/mods on a test server that I set up and did it on a clean build with a new world. The Command block still was not not responsive to a button press with the command `/tpj @p 12`. Other commands such as `/tp @p 0 0 0` were working, however.

**Comment by maruohon** [comment]
Hmm, it should be giving you at least some output in the command block output field? What about if you just make a typo, like `/tpjj @p 12`, does it then at least complain about not finding the command?

**Comment by sjaustad** [comment]
No actually it doesn't give output for any commands, even one's that work. 
![image](https://user-images.githubusercontent.com/33435811/61149980-9babe580-a49f-11e9-86d1-55338a71ab84.png)

**Comment by maruohon** [comment]
Do you have the output tracking enabled in the command block? The button next to it should show `O` when it's enabled, and `X` when it's not.

**Comment by sjaustad** [comment]
Yeah I realized that which is why I got rid of the screen shot. I got it figured out though. It was showing the output for a `/tp real_user` command block or `/tp non-existent_user` with a success or error respectively. With both `/tpj` and `/tpjj` it didn't do anything. I also loaded up a clean server build and I removed Towny, Luckperm, and Forge Essentials (the only permission mods that I have) and it still didn't work.

**Comment by maruohon** [comment]
It still doesn't show anything for the tpj commands? How is that even possible... Does it work if you remove the Bukkit whatever component you use, and just run it under normal Forge?

**Comment by LemADEC** [comment]
Forge provides its own `/forge setdimension <entity> <world> [<x> <y> <z>]`, give a try maybe?
